src/Behat/MinkExtension/ServiceContainer/Driver/Selenium2Factory.php

```diff
@@ -93,6 +93,7 @@ protected function getCapabilitiesNode()
             ->addDefaultsIfNotSet()
             ->normalizeKeys(false)
             ->children()
+                ->scalarNode('unexpectedAlertBehaviour')->defaultValue('ignore')->end()
```

Comment:
I suggest we put no default value, as done for others. We rely on Selenium's default for most of them (especially given that different implementations might support different ones)

Reply:
OK, makes sense, I will change it.
Thanks for the review :)

What about the Travis failures below? They don't seem to be related to my change?